src/commands/synthetics/deploy-tests-lib.ts

```diff
@@ -5,6 +5,26 @@ import {DeployTestsCommandConfig, LocalTriggerConfig, MainReporter, ServerTest}
 import {getTestConfigs} from './test'
 import {isLocalTriggerConfig} from './utils/internal'
 
+const removeExcludedFields = (
```

Comment:
That's fine for our demo, but I'd prefer this to support JSON path selectors. For example, the default would become `config.*`, and users could change it to `config.request.url` if they want to be very specific. 

There must be libraries that handle this type of object merge with excluded fields already